circleci/resource_circleci_environment_variable.go

```diff
@@ -168,3 +254,37 @@ func getOrganization(d *schema.ResourceData, providerClient *ProviderClient) str
 
 	return providerClient.organization
 }
+
+func getEnvironmentVariableId(d *schema.ResourceData) error {
```

Comment:
nit: I would change the name of this func given that it has side effects (the three sets)

Reply:
I've updated the name to be `setOrgProjectNameFromEnvironmentVariableId` which makes more sense